src/components/Button/LinkButton.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import SurveyButton from '.';
+
+function LinkButton({ linkTo = '/', label }) {
```

Comment:
Comment to the whole project -> Define `isRequired` or default value for every property.

Reply:
Define some `propTypes` for every component. Project is lacking the data transparency. It is not obvious what data is passed to component.